variants/heltec_esp32c3/variant.h

```diff
@@ -1,16 +1,43 @@
-#define BUTTON_PIN 9
-
-// LED pin on HT-DEV-ESP_V2 and HT-DEV-ESP_V3
-// https://resource.heltec.cn/download/HT-CT62/HT-CT62_Reference_Design.pdf
-// https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V3_Sch.pdf
-#define LED_PIN 2 // LED
-#define LED_INVERTED 0
+/*
+ * Heltec HT-CT62, Heltec HT-DEV-ESP V2 and Heltec HT-DEV-ESP V3
+ * Heltec HT-CT62 Schematic: https://resource.heltec.cn/download/HT-CT62/HT-CT62_Reference_Design.pdf
+ * Heltec HT-DEV-ESP V3 Schematic: https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V2_Sch.pdf
+ * Heltec HT-DEV-ESP V3 Schematic: https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V3_Sch.pdf
+ */
 
 #define HAS_SCREEN 0
+
+// Board does not have enough free pins for a UART GPS, use an I2C GPS
 #define HAS_GPS 0
 #undef GPS_RX_PIN
 #undef GPS_TX_PIN
 
+// USER_SW (HT-DEV-ESP V2/V3)
+#define BUTTON_PIN 9
+
+/*
+ * LED (HT-DEV-ESP_V2/V3)
+ * HT-CT62 reference schematic connects the LED to GPIO18 but HT-DEV-ESP schematic connects the LED to GPIO2
+ */
+#define LED_PIN 2
+#define LED_INVERTED 0
+
+/*
+ * I2C
+ * Not defined on schematics, but these are the only free GPIO exposed on the castellated pads.
+ * There are pin mux conflicts: GPIO18 = USB_DN and GPIO19 = USB_DP. Since the HT-DEV-ESP provides a CP2102, re-use these ports
+ * for I2C instead since it's more useful (attaching peripherals vs. connecting to a host PC). Boards which use the native ESP32
+ * USB will require a separate variant (refer to Heltec HRU-3601 variant for example). Perhaps a "heltec_esp32c3_usb" variant
+ * would make things easier?
+ */
+#define I2C_SCL 18
+#define I2C_SDA 19
+
```

Comment:
What about the original Pins 0 and 1?

Reply:
> What about the original Pins 0 and 1?

These are used for a 32kHz crystal oscillator on the HT-DEV-ESP32 and reference design, https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V3_Sch.pdf and https://resource.heltec.cn/download/HT-CT62/HT-CT62_Reference_Design.pdf respectively

Reply:
As commented in the other thread, GPIO0 and GPIO1 can be freed up for HT-DEV-ESP users by desoldering the 32kHz crystal and passives. I don't think this crystal provides any benefit to Meshtastic anyway; the HT-CT62 module has a 40MHz crystal onboard for the ESP32-C3 already.

The 32kHz crystal is just an extra clock source and desoldering it won't affect HT-DEV-ESP board functionality. And anyone building a Meshtastic device around the HT-CT62 is not likely to bother with adding a 32kHz crystal either.

So this is another option we can consider too.